src/main/java/org/fundacionjala/coding/christian/CesarCrypt.java

```diff
@@ -0,0 +1,62 @@
+package org.fundacionjala.coding.christian;
+
+import static java.lang.Character.isUpperCase;
+
+/**
+ * Class CesarCrypt.
+ */
+public class CesarCrypt {
```

Comment:
It is missing the vigenere cipher
also you should implement the encode and decode.